src/meta/processors/indexMan/RebuildEdgeIndexProcessor.cpp

```diff
@@ -0,0 +1,109 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "meta/ActiveHostsMan.h"
+#include "meta/processors/admin/AdminClient.h"
+#include "meta/processors/indexMan/RebuildEdgeIndexProcessor.h"
+
+namespace nebula {
+namespace meta {
+
+void RebuildEdgeIndexProcessor::process(const cpp2::RebuildIndexReq& req) {
+    auto space = req.get_space_id();
+    const auto &indexName = req.get_index_name();
+
+    LOG(INFO) << "Rebuild Edge Index Space " << space << ", Index Name " << indexName;
+
+    folly::SharedMutex::ReadHolder rHolder(LockUtils::edgeIndexLock());
+    std::unique_ptr<AdminClient> client(new AdminClient(kvstore_));
+    auto spaceKey = MetaServiceUtils::spaceKey(space);
+    auto spaceRet = doGet(spaceKey);
+    if (!spaceRet.ok()) {
+        LOG(ERROR) << "Space " << space << " Not Found";
+        resp_.set_code(cpp2::ErrorCode::E_NOT_FOUND);
+        onFinished();
+        return;
+    }
```

Comment:
Could we using spaceExist method at here?

Reply:
`CHECK_SPACE_ID_AND_RETURN` is better